website/docs/r/policy_security_policy_rule.html.markdown

```diff
@@ -0,0 +1,98 @@
+---
+subcategory: "Firewall"
+layout: "nsxt"
+page_title: "NSXT: nsxt_policy_security_policy_rule"
+description: A resource to configure a Security Policy Rule.
+---
+
+# nsxt_policy_security_policy_rule
+
+This resource provides a method for the management of Security Policy Rule.
+
+Note: to avoid unexpected behavior, don't use this resource and resource `nsxt_policy_security_policy` to manage rules under a security policy at the same time. 
+Instead, please use this resource with resource `nsxt_policy_parent_security_policy` to manage a security policy and its rules separately, and use `nsxt_policy_security_policy` to manage a security policy and its rules in one single resource.
+Highly recommend to add `depends_on` to ensure its reference `nsxt_policy_parent_security_policy` could be consumed first.
```

Comment:
Can you please clarify why explicit dependency is needed?

Reply:
My bad. Before I misunderstood implicit dependency only works while creating resources not updating. Removed it.